MastodonSDK/Sources/MastodonUI/View/Content/MediaView.swift

```diff
@@ -77,6 +73,20 @@ public final class MediaView: UIView {
         return label
     }()
     
+    let _altViewController: UIViewController! = {
+        if #available(iOS 15.0, *) {
```

Comment:
I'm so looking forward to when we can get rid of these checks.

But in these case it means: Alt-text is only available for >iOS 15, right?

Reply:
Correct.